### PR TITLE
Add links to teachers' day and Code4Research to homepage

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

File: content/home.md [hunk 1]
```diff
@@ -13,9 +13,10 @@ September.
 
 We&rsquo;ll be putting on a programme of talks, workshops, and other events
```
Comment (snim2) [inline content/home.md:14]: Can use a plain apostrophe

File: content/home.md [hunk 2]
```diff
@@ -13,9 +13,10 @@ September.
...
 aimed at the whole Python community, from web developers to hardware engineers,
-from professionals to hobbyists.  There will be an education programme for
-teachers, and an expanded research programme aimed at researchers from the
-humanities, arts and scientists.
+from professionals to hobbyists.  There will be an [education
+programme](/teachers/) for teachers, and an [expanded research
+programme](/research/) aimed at researchers from the humanities, arts and
+scientists.
 
 We&rsquo;re a community&hyphen;led, volunteer&hyphen;run conference, and this
```
Comment (snim2) [inline content/home.md:21]: More apostrophies

**Comment by snim2** [comment]
@inglesp Do you want to deal with those unicode issues here or in another PR?

**Comment by meshy** [comment]
@snim2 I was going to look at them tonight in https://github.com/PyconUK/2016.pyconuk.org/issues/14

**Comment by snim2** [comment]
OK, cool

**Pull request merged**